readme.md

```diff
@@ -222,7 +222,7 @@ npx babel-upgrade --write --install
 - Maybe the version should just reflect the version that it targets?
 
 ## Development
-```sh
-$ npm i
-$ npm start
+```Shell
```

Comment:
why did you change the language here? Does `Shell` work in GitHub?

Reply:
It does work. but so does sh.

Reply:
okay, cool